card/lib/card/content.rb

```diff
@@ -134,85 +152,99 @@ def parse_content content
       ALLOWED_TAGS['table'] += %w[ cellpadding align border cellspacing ]
     end
 
-    ALLOWED_TAGS.each_key {|k|
+    ALLOWED_TAGS.each_key do |k|
       ALLOWED_TAGS[k] << 'class'
       ALLOWED_TAGS[k] << 'style' if Card.config.allow_inline_styles
       ALLOWED_TAGS[k]
-    }
+    end
     ALLOWED_TAGS
 
-    ATTR_VALUE_RE = [ /(?<=^')[^']+(?=')/, /(?<=^")[^"]+(?=")/, /\S+/ ]
+    ATTR_VALUE_RE = [/(?<=^')[^']+(?=')/, /(?<=^")[^"]+(?=")/, /\S+/]
 
     class << self
-
       ## Method that cleans the String of HTML tags
       ## and attributes outside of the allowed list.
 
       # this has been hacked for card to allow classes if
       # the class begins with "w-"
-      def clean!( string, tags = ALLOWED_TAGS )
-        string.gsub( /<(\/*)(\w+)([^>]*)>/ ) do
+      def clean!(string, tags=ALLOWED_TAGS)
```

Comment:
Assignment Branch Condition size for clean! is too high. [25.67/15]<br>Method has too many lines. [30/15]<br>Use def without parentheses.